### PR TITLE
Add clipUtils

### PR DESCRIPTION
**Pull request by sachinprasadhs**
- Added Clip models to clip utils.
- Verified numerics for SDCLIP and SDXLClipG, ClipTextModel here 
- https://colab.research.google.com/drive/1owGlSRtXZFCPsPVFYGzD6yQYfMn_DbyS?resourcekey=0-to0rWWTug0voaXGh2bHnpg#scrollTo=YzGP-K_6tuWG
- https://colab.sandbox.google.com/drive/1R2hYaCc2k1l40MLpBFIvtRze4suDrRee?resourcekey=0-JtftJsmb8NofQIWZ7eqg3g#scrollTo=2-WDk4Gj6FCT

### CODE REVIEW

**Review by divyashreepathihalli — APPROVED** [review]

**Review by divyashreepathihalli — CHANGES_REQUESTED** [review]
The imports needs to be fixed

File: keras_cv/src/models/stable_diffusion_v3/MMDit.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-from keras_cv.backend import keras
+from keras_cv.src.backend import keras
 from keras_cv.layers.vit_layers import PatchingAndEmbedding
 from keras_cv.models.stable_diffusion.v3 import embedding
```
Comment (divyashreepathihalli) [inline keras_cv/src/models/stable_diffusion_v3/MMDit.py:3]: This import will need to be updated to `from keras_cv.src.models.stable_diffusion.v3 import embedding`, etc

Comment (divyashreepathihalli) [reply keras_cv/src/models/stable_diffusion_v3/MMDit.py:3]: same with all files

File: keras_cv/src/models/stable_diffusion_v3/clip_utils_test.py [hunk 2]
```diff
@@ -91,8 +96,8 @@ def test_sd3_tokenizer(self):
         out_keras = sd3_tokenizer.tokenize_with_weights("A cat")
         expected_g_tokens = [(49406, 1.0), (320, 1), (2368, 1), (49407, 1.0)]
         expected_l_tokens = [(49406, 1.0), (320, 1), (2368, 1), (49407, 1.0)]
-        expected_t5xxl_tokens = [(71, 1), (1712, 1), (1, 1.0), (0, 1.0)]
+        expected_t5xxl_tokens = [(71, 1), (1712, 1), (1, 1.0), (0, 1.0)] # pylint: disable=unused-variable
```
Comment (divyashreepathihalli) [inline keras_cv/src/models/stable_diffusion_v3/clip_utils_test.py:99]: two spaces before addinf this ignore lint comment

Comment (sachinprasadhs) [reply keras_cv/src/models/stable_diffusion_v3/clip_utils_test.py]: Added # noqa: F841

**Pull request merged**